src/DotLiquid/Strainer.cs

```diff
@@ -142,19 +142,8 @@ public object Invoke(string method, List<object> args)
                 if (args[argumentIndex] is IConvertible convertibleArg)
                 {
                     var parameterType = parameterInfos[argumentIndex].ParameterType;
-                    if (convertibleArg.GetType() != parameterType
-                        && !parameterType
-#if NETSTANDARD1_3
-                            .GetTypeInfo()
-#endif
-                            .IsAssignableFrom(
-                                convertibleArg
-                                    .GetType()
-#if NETSTANDARD1_3
-                                    .GetTypeInfo()
-#endif
-                                    )
-                        )
+                    if (convertibleArg.GetType() != parameterType &&
+                        !parameterType.IsInstanceOfType(convertibleArg))

```

Comment:
OK - I did change this line of code from `IsAssignableFrom` to `IsInstanceOfType`. I'll change it back, just so there's no code changes at all in this PR.